lib/events.js

```diff
@@ -19,19 +27,26 @@ module.exports = function (pinoInstance, internalExtremeHandler) {
     internalExtremeHandler()
   }
 
+  process.on('beforeExit', wrap('beforeExit'))
```

Comment:
Do we need `beforeExit`? It's only called during normal shutdown not on an explicit shutdown. And the `exit` event is still going to fire after it.